### PR TITLE
[WIP] Add support for Miniscope-DAQ-V3 and MiniscopeAnalysis.  Add package version.  Update readme.

### PR DESCRIPTION
**Pull request by kabilar**

### CODE REVIEW

File: elements_imaging/imaging.py [hunk 1]
```diff
@@ -632,7 +734,7 @@ class ActivityExtractionMethod(dj.Lookup):
     extraction_method: varchar(32)
     """
 
-    contents = zip(['suite2p_deconvolution', 'caiman_deconvolution', 'caiman_dff'])
+    contents = zip(['suite2p_deconvolution', 'caiman_deconvolution', 'caiman_dff', 'miniscope_analysis'])
```
Comment (ttngu207) [inline elements_imaging/imaging.py:737]: perhaps you mean to add `miniscope_analysis_deconvolution` and `miniscope_analysis_dff` here?

**Review by ttngu207 — CHANGES_REQUESTED** [review]
Overall looks good, but seems not ready to be merged as there are a number of place-holder (commented as TODOs) in the ingestion routines.
I'd suggest tagging this PR as Work In Progress `[WIP]` and we can review again once it's ready

File: elements_imaging/imaging.py (hunk 1 above)
Comment (kabilar) [reply elements_imaging/imaging.py:737]: Thank you.  Fixed.

**Review by dimitri-yatsenko — APPROVED** [review]

**Comment by kabilar** [comment]
Will open this pull request on `element-miniscope`.

**Pull request closed without merging**